app/helpers/pagination_helper.rb

```diff
@@ -0,0 +1,17 @@
+module PaginationHelper
+  def paginate(total_items, items_per_page, current_page, base_url)
+    total_pages = (total_items.to_f / items_per_page).ceil
+    pagination_html = '<ul class="pagination">'
+
+    if current_page > 1
+      pagination_html += '<li>' + link_to('Previous', "#{base_url}?page=#{current_page - 1}") + '</li>'
```

Comment:
**[standard]** <Style/StringLiterals> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
```suggestion
      pagination_html += '<li>' + link_to('Previous', "#{base_url}?page=#{current_page - 1}") + "</li>"
```